web/scripts/api.js

```diff
@@ -216,6 +233,39 @@ class ComfyApi extends EventTarget {
 		}
 	}
 
+	/**
+	 * Gets prompts saved by the user
+	 * @returns Saved prompts with outputs
+	 */
+	async getSaved() {
+		// note that this could, and maybe should, be persisted
```

Comment:
was considering trying to implement this feature myself, and my plan was to manage it like the images. new folder added to `.gitignore`, './saved-workflows/` or something, and the API just lets you browse that folder. each workflow can be saved as its own `.json` file.

history definitely makes sense in localstorage though.